pom.xml

```diff
@@ -56,6 +57,26 @@
       into a fat executable jar. This is a jar that contains all the dependencies
       required to run it, so you can just run it with java -jar -->
     <plugins>
+      <!-- JAXB Maven Plugin-->
+      <plugin>
```

Comment:
consider doing this the same way as in mod-oai-pmh... having consistency between the projects will make things easier in the long run.  

https://github.com/folio-org/mod-oai-pmh/blob/5a3e214abcca56d9c2f8bb5fa50ea5f2f3c86924/pom.xml#L157

it's also nice to have the fluentAPI and jaxb annotations.  I'm not sure you get those this way.